### PR TITLE
Run E2E tests in latest FLC version environments

### PR DESCRIPTION
**Pull request by albertogdd**
## Description

Update FLC environments where we run our E2E tests. Now, we use the latest version ones.

## How can this be tested?

https://github.com/Dynatrace/dynatrace-operator/actions/runs/8170692777

## Checklist

~~- [ ] Unit tests have been updated/added~~
- [X] PR is labeled accordingly with a single label
- [X] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: .github/workflows/e2e-tests.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: E2E tests
 
 on:
   schedule:
-    # every work day at 5:00 AM
+    # every work day at 00:00 AM
```
Comment (chrismuellner) [inline .github/workflows/e2e-tests.yaml:5]: should we just say midnight? what time zone?

File: .github/workflows/e2e-tests.yaml [hunk 2]
```diff
@@ -13,11 +13,11 @@ on:
...
 jobs:
   run-in-k8s:
-    name: Run in Kubernetes 1.26 (${{ github.event.inputs.target }})
+    name: Run in Kubernetes latest (${{ github.event.inputs.target }})
     environment: E2E
     env:
-      FLC_NAMESPACE: dto-k8s-ondemand
-      FLC_ENVIRONMENT: dto-k8s-1-26
+      FLC_NAMESPACE: dto-daily
+      FLC_ENVIRONMENT: dto-k8s-latest-flc
```
Comment (chrismuellner) [inline .github/workflows/e2e-tests.yaml:20]: somewhat unrelated to this specific PR, but is there a reason we are not using a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-using-a-single-dimension-matrix) to reuse the steps? It seems only the `FLC_ENVIRONMENT` is different between `run-in-k8s` and `run-in-ocp` jobs

**Review by chrismuellner — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e-tests.yaml (hunk 2 above)
Comment (albertogdd) [reply .github/workflows/e2e-tests.yaml:20]: I just wrote a user story today about this, you can check it in the epic (I have no access to Jira now)

File: .github/workflows/e2e-tests.yaml (hunk 1 above)
Comment (albertogdd) [reply .github/workflows/e2e-tests.yaml:5]: 00:00 is more clear to me than midnight, just IMO
I will add the timezone 👍

Comment (albertogdd) [reply .github/workflows/e2e-tests.yaml]: Updated

**Review by chrismuellner — APPROVED** [review]

**Pull request merged**